tests/test_simulation.py

```diff
@@ -1,10 +1,27 @@
+import time
+
 from config import API_KEY, HOST, WILDFIRE_SCENARIO_ID
 
 from sedaro import SedaroApiClient
 
 sedaro = SedaroApiClient(api_key=API_KEY, host=HOST)
 
 
+def _job_status_is_pending(job):
+    return job['status'] == 'PENDING'
+
+
+def _wait_until_passed_pending(simulation_handle):
+    start_time = time.perf_counter()
+    print('- Wait until no longer "PENDING"...')
+
+    while _job_status_is_pending(simulation_handle.status()):
+        max_seconds = 180
```

Comment:
I had to allow for <= 180 seconds of 'PENDING' before trying to move on. Is this just because of my computer? Or is it really taking that long to deploy? @baswelsh

It takes this long when I start a sim from the UI as well.

Reply:
Hmm something in the new architecture might be taking longer.  How long did it previously take on your machine?